spec/features/user_past_order_view_spec.rb

```diff
@@ -1,36 +1,50 @@
 require 'rails_helper'
 
 describe 'users past order views', type: :feature do
-  let(:users) { Fabricate(:users) }
+  let(:user) { Fabricate(:user) }
+  let(:category) { Fabricate(:category) }
+  let(:item) { item = Fabricate.build(:item)
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.